content/en/docs/ops/deployment/requirements/index.md

```diff
@@ -48,43 +52,39 @@ requirements:
     - The `version` label: This label indicates the version of the application
       corresponding to the particular deployment.
 
-- **Application UIDs**: Ensure your pods do **not** run applications as a user
-  with the user ID (UID) value of **1337**.
-
-- **`NET_ADMIN` and `NET_RAW` capabilities**: If your cluster enforces pod security policies,
-  they must allow injected pods to add the `NET_ADMIN` and `NET_RAW` capabilities.
-  If you use the [Istio CNI Plugin](/docs/setup/additional-setup/cni/),
-  this requirement no longer applies. To learn more about the `NET_ADMIN` and `NET_RAW`
-  capabilities, see [Required pod capabilities](#required-pod-capabilities), below.
+- **Named service ports**: Service ports may optionally be named to explicitly specify a protocol.
+  See [Protocol Selection](/docs/ops/configuration/traffic-management/protocol-selection/) for
+  more details.
 
 ## Ports used by Istio
 
-The following ports and protocols are used by Istio. Ensure that there are no
-TCP headless services using a TCP port used by one of Istio's services.
+The following ports and protocols are used by Istio.
 
 | Port | Protocol | Used by | Description |
 |----|----|----|----|
-| 8060 | HTTP | Citadel | GRPC server |
-| 8080 | HTTP | Citadel agent | SDS service monitoring |
-| 9090 | HTTP |  Prometheus | Prometheus |
-| 9091 | HTTP | Mixer | Policy/Telemetry |
-| 9876 | HTTP | Citadel, Citadel agent |  ControlZ user interface |
-| 9901 | GRPC | Galley| Mesh Configuration Protocol |
 | 15000 | TCP | Envoy | Envoy admin port (commands/diagnostics) |
 | 15001 | TCP | Envoy | Envoy Outbound |
 | 15006 | TCP | Envoy | Envoy Inbound |
-| 15004 | HTTP | Mixer, Pilot | Policy/Telemetry - `mTLS` |
-| 15010 | HTTP | Pilot | Pilot service - XDS pilot - discovery |
-| 15011 | TCP | Pilot | Pilot service - `mTLS` - Proxy - discovery |
-| 15014 | HTTP | Citadel, Citadel agent, Galley, Mixer, Pilot, Sidecar Injector | Control plane monitoring |
-| 15020 | HTTP | Ingress Gateway | Pilot health checks |
+| 15020 | HTTP | Envoy | Health checks |
+| 15090 | HTTP | Envoy | Prometheus telemetry |
+| 15010 | GRPC | Istiod | XDS and CA services (plaintext) |
```

Comment:
Need to verify `Istiod` spelling and add to the .spelling file.

Reply:
Per a discussion, we should capitalize when talking about the component and not when when talking about the deployment, so this should be good to go.